eotsmanager/localmanager.go

```diff
@@ -197,12 +205,40 @@ func (lm *LocalEOTSManager) SignSchnorrSig(fpPk []byte, msg []byte, passphrase s
 		return nil, fmt.Errorf("failed to get EOTS private key: %w", err)
 	}
 
+	return lm.SignSchnorrSigFromPrivKey(privKey, fpPk, msg)
+}
+
+// SignSchnorrSigFromPrivKey signs a with schnorr using the private key and updating metrics by the fpPk
+func (lm *LocalEOTSManager) SignSchnorrSigFromPrivKey(privKey *btcec.PrivateKey, fpPk []byte, msg []byte) (*schnorr.Signature, error) {
```

Comment:
```suggestion
// signSchnorrSigFromPrivKey signs a Schnorr signature using the private key and updates metrics by the fpPk
func (lm *LocalEOTSManager) signSchnorrSigFromPrivKey(privKey *btcec.PrivateKey, fpPk []byte, msg []byte) (*schnorr.Signature, error) {
```
This probably can be private

Reply:
move to private https://github.com/babylonchain/finality-provider/pull/309/commits/16d3d195a1ba139faedd65856aac7f39caff8136